extensions-core/avro-extensions/src/test/java/org/apache/druid/data/input/avro/AvroFlattenerMakerTest.java

```diff
@@ -78,7 +78,8 @@ public void getRootField()
         flattener.getRootField(record, "someNull")
     );
     Assert.assertEquals(
-        record.getSomeFixed(),
+        // Casted to an array by transformValue
+        record.getSomeFixed().bytes(),
```

Comment:
What data type is "someFixed"? This code is expecting it to be translated as a `byte[]`. I think most downstream users of these parsed datums will be interpreting them using Rows.objectToStrings, which is going to translate a `byte[]` into a base64 encoded string. Is that sensible behavior?

Would you be able to add some documentation to this page: https://druid.apache.org/docs/latest/ingestion/data-formats.html#avro-ocf about how the various Avro types that we support will be handled?

Reply:
Yes so `fixed` and `bytes` are encoded equivalently in Avro, `fixed` just has an enforced size. There is a `binaryAsString` option for decoding these as strings if need be. It would be nice to support that as a per-field transformation though rather than applying to all binary types as you may only want some to be interpreted as strings and others to be returned verbatim.

If docs were to be added I would probably put them on the `druid-avro-extensions` page and link to them from the various places Avro parsing it used.